src/util/EventManager.ts

```diff
@@ -0,0 +1,25 @@
+
+class EventManager {
+
+    public trigger(eventName: string) {
+        const event = new Event(eventName);
+        document.dispatchEvent(event);
+    }
+
+    // TODO: use eventName: type
```

Comment:
Why not add typings already now, that is practically one line?

Reply:
I had to do this in this task but somehow forgot, good that you noticed